server_integration_test.go

```diff
@@ -567,6 +604,41 @@ func TestServerGet(t *testing.T) {
 	}
 }
 
+func TestServerRelativeGet(t *testing.T) {
+	tmpWorkingDir, err := ioutil.TempDir("", "sftp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpWorkingDir)
+
+	listenerGo, hostGo, portGo := testServer(t, GOLANG_SFTP, READONLY, tmpWorkingDir)
+	defer listenerGo.Close()
+
+	fn := randName()
+	tmpFileLocal := filepath.Join("/tmp", fn)
```

Comment:
You're absolutely correct. This code was essentially copied from the 8 existing occurrences in the test. Would you like a PR, separate commit, or something else to correct this issue?